assets/sass/components/analytics-4/audience-segmentation/_googlesitekit-audience-selection-panel.scss

```diff
@@ -207,6 +207,50 @@
 				width: initial;
 			}
 		}
+
+		.googlesitekit-audience-creation-error-notice {
+			align-items: center;
+			background-color: $c-content-on-primary;
+			border: 2px solid $c-utility-error;
+			border-radius: $br-sm;
+			display: flex;
+			gap: 16px;
+			margin-bottom: 12px;
+			padding: 14px $grid-gap-phone;
+
+			svg {
+				color: $c-utility-error;
+				flex-shrink: 0;
+			}
+
+			p.googlesitekit-audience-creation-error-notice__title {
+				color: $c-surfaces-on-surface;
+				font-size: $fs-body-md;
+				font-weight: $fw-medium;
+				line-height: $lh-body-md;
+				margin: 0;
+			}
+
+			p.googlesitekit-audience-creation-error-notice__description {
+				color: $c-surfaces-on-surface;
+				font-size: $fs-body-sm;
+				font-weight: $fw-normal;
+				letter-spacing: $ls-xs;
+				line-height: $lh-body-sm;
+				margin: 0;
+			}
+
+			.googlesitekit-audience-creation-error-notice__actions {
+
+				.mdc-button {
+					font-weight: $fw-medium;
+					gap: 10px;
```

Comment:
Curious to know, what is the purpose of this `gap` style in this context. Thanks!

Reply:
Removed, thanks!